src/modules/StorageProposal.rb

```diff
@@ -668,7 +668,7 @@ def try_add_boot(conf, disk, force)
         pb = {}
 	pb["mount"] = Partitions.BootMount
 	pb["size"] = Partitions.ProposedBootsize
-	if disk.fetch("label","")=="gpt" && !Partitions.EfiBoot
+	if disk.fetch("label","")=="gpt" && !Partitions.EfiBoot && !is_dasd?(disk.fetch("device", ""))
```

Comment:
It also confuses me and I also think the code is wrong: I suppose ```sz``` should be ```cyl_size / 1024``` instead of ```- 1024```. But I don't want to change more than required.